tests/tests_app/core/test_lightning_api.py

```diff
@@ -85,36 +84,6 @@ def test_app_state_api(runtime_cls):
     os.remove("test_app_state_api.txt")
 
 
-class A2(LightningFlow):
-    def __init__(self):
-        super().__init__()
-        self.var_a = 0
-        self.a = _A()
-
-    def update_state(self):
-        state = AppState()
-        # this would download and push data to the REST API.
-        assert state.a.work_a.var_a == 0
-        assert state.var_a == 0
-        state.var_a = -1
-
-    def run(self):
-        if self.var_a == 0:
-            self.update_state()
-        elif self.var_a == -1:
-            self._exit()
-
-
-# TODO: Find why this test is flaky.
-@pytest.mark.skip(reason="flaky test.")
-@pytest.mark.parametrize("runtime_cls", [SingleProcessRuntime])
```

Comment:
Did you try MultiProcessRuntime for this test ?

Reply:
Added back